documentation/blog/2025-09-15-subrecipes-in-goose/index.md

```diff
@@ -0,0 +1,415 @@
+
+Network calls, file operations, and API calls can fail. Add retry logic with proper cleanup:
+
+```yaml
+retry:
+  max_retries: 3
+  checks:
+    - type: shell
+      command: test -f "expected_output.txt"
+  on_failure: rm -f "partial_output.txt"
+  timeout_seconds: 60
+```
+
+### 5. Monitor Resource Usage
+
+When running multiple subrecipes in parallel, watch your API rate limits and system resources. You might need to adjust the execution strategy for resource-intensive tasks.
+
+## More Subrecipe Workflows
+
+Start simple. Pick a complex task you do regularly and break it into 2-3 smaller pieces. Create individual recipes for each piece, test them separately, then build a parent recipe to orchestrate them.
+
+Some ideas to get you started:
+
+- **Content Creation Pipeline**: Research, writing, editing, and formatting
+- **Development Workflow**: Code generation, testing, documentation, deployment
+- **Data Processing**: Collection, cleaning, analysis, visualization
+- **Project Setup**: Structure creation, configuration, initial files, documentation
+
+## The Future of Orchestrated AI
+
+Subrecipes represent something bigger than just a goose feature. They're a glimpse into how we'll work with AI in the future -- not as monolithic systems trying to do everything, but as specialized agents working together toward common goals.
+
+Each recipe becomes a reusable component that you can mix and match. Build a library of specialized recipes, then combine them in different ways for different projects. It's like having a toolkit of AI specialists ready to tackle any challenge.
+
+Ready to start building your own subrecipe workflows? The kitchen is open, and all the ingredients are waiting for you.
+
+## Share Your Recipes with Us!
+
+Make a [pull request to goose](https://github.com/block/goose) to include your recipe ideas with other goose users! Add them to the `documentation/src/pages/recipes/data/recipes` path in the goose GitHub repo.
+
+---
+
+## FAQ
+
+**Q: Can subrecipes call other subrecipes?**
+A: No, a recipe running as a subrecipe cannot itself call other subrecipes. This keeps the workflow manageable and avoids deep nesting or looping recipes ending in infinite recursion.
+
+**Q: How do I debug a failing subrecipe workflow?**
+A: Test each subrecipe individually first, check parameter passing, and use the retry logic with proper checks.
+
+**Q: Can I use different AI models in different subrecipes?**
+A: Absolutely! This is one of the main benefits -- use the best model for each specific task.
+
+**Q: What happens if one subrecipe fails?**
+A: The parent recipe will typically fail too, unless you build in error handling. Always test individual recipes first.
+
+**Q: How do I share data between subrecipes?**
+A: Use the template variable `{{ recipe_dir }}` within each recipe's instructions to share filenames. Just be careful of concurrent writes to the same files if you have some subrecipes running in parallel.
```

Comment:
This is good but i'm wondering if this information should be in our documentation. Then we can link to that doc in your blog.